doc/ref_model/chapters/chapter01.md

```diff
@@ -38,18 +38,18 @@ This section introduces the high-level principles of infrastructure abstraction
    - Networking resources (Limited to connectivity services only)
    - Acceleration resources
 1. Cloud Infrastructure exposed resources should be supplier independent
-1. All Cloud Infrastructure Application Programming Interfaces (APIs) must ensure Interoperability (multi-vendor, components substitution), drive Simplification, and open source implementations that have an open governance model (e.g. come from Open Communities or Standards Development Organisations). Through such APIs will cloud infrastructure resources be discovered/monitored by management entities, configured on behalf of VNFs and consumed by VNFs.
-1. VNFs should be modular and be designed to utilise the minimum resources required for the service
+1. All Cloud Infrastructure Application Programming Interfaces (APIs) must ensure Interoperability (multi-vendor, components substitution), drive Simplification, and open source implementations that have an open governance model (e.g. come from Open Communities or Standards Development Organisations). Through such APIs will cloud infrastructure resources be discovered/monitored by management entities, configured on behalf of workloads and consumed by workloads.
+1. Workloads should be modular and be designed to utilise the minimum resources required for the service
 1. Cloud Infrastructure shall support pre-defined and parameterised sizes
```

Comment:
sizes of what?

Reply:
This part has been removed from the chapter.